src/vs/workbench/contrib/webview/common/resourceLoader.ts

```diff
@@ -83,7 +83,7 @@ function normalizeRequestPath(requestUri: URI) {
 
 	// Modern vscode-resources uris put the scheme of the requested resource as the authority
 	if (requestUri.authority) {
-		return URI.parse(requestUri.authority + ':' + requestUri.path);
+		return URI.parse(requestUri.authority + ':' + encodeURIComponent(requestUri.path).replace(/%2F/g, '/'));
```

Comment:
Which operating system did you test this on. The change looks good but I'm not sure if it will work on windows

Reply:
Ubuntu 18.04

Reply:
What happens for windows style paths though (ones that use `\`)?

Reply:
I don't have a windows VM at hand to be able to build from sources and test it but looking at the uri.ts file it seems that `\` characters are converted to `/` before constructing the uri object
https://github.com/microsoft/vscode/blob/ab147f8b8e2af23fd61385b563c246847bc50f2b/src/vs/base/common/uri.ts#L306-L308

and converted back again with the [fsPath getter](https://github.com/microsoft/vscode/blob/ab147f8b8e2af23fd61385b563c246847bc50f2b/src/vs/base/common/uri.ts#L201)